about.css

```diff
@@ -0,0 +1,465 @@
+  background-repeat: no-repeat;
+  background-size: cover;
+}
+
+.section-2 .event-2 {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  gap: 20px;
+  background:
+    linear-gradient(
+      0deg,
+      rgba(232, 117, 107, 0.687),
+      rgba(233, 74, 74, 0.664)
+    ),
+    url('images/pexels-matheus-bertelli-3321802.jpg');
+  background-repeat: no-repeat;
+  background-size: cover;
+}
+
+/* Footer Section */
+
+.footer {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: center;
+  gap: 30px;
+  padding: 50px 5vw;
+  background: #272a31;
+  color: #9b9b9b;
+}
+
+.footer .logo {
+  height: 30px;
+  width: 160px;
+  filter: brightness(0) invert(1);
+}
+
+.footer .text-inner-container {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  gap: 5px;
+  white-space: nowrap;
+  font-size: 13px;
+}
+
+.footer .licence {
+  display: none;
+}
+
+/* Desktop */
+@media only screen and (min-width: 768px) {
+  .header-section1-container {
+    background-image: none;
+    padding: 0;
+  }
+
+  /* Header */
+
+  .header .hamburger {
+    display: none;
+  }
+
+  .header .desktop-nav {
+    display: flex;
+    flex-direction: column;
+  }
+
+  .header .logo {
+    height: 40px;
+    width: 105px;
+  }
+
+  .header .desktop-nav .black-row {
+    display: flex;
+    justify-content: end;
+    align-items: center;
+    gap: 20px;
+    background: #272a31;
+    padding: 10px 15vw;
+    color: white;
+    font-size: 10px;
+  }
+
+  .header .desktop-nav .white-row {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    background: white;
+    padding: 15px 15vw;
+  }
+
+  .header .desktop-nav .white-row .list {
+    display: flex;
+    gap: 20px;
+  }
+
+  .header .desktop-nav .white-row .a {
+    color: #9d9d9d;
+    font-size: 15px;
+  }
+
+  .header .desktop-nav .white-row .a:hover {
+    color: #ec5242;
+  }
+
+  .header .desktop-nav .white-row .a-cc {
+    border: solid 4px red;
+    padding: 10px;
+    color: #ec5242;
+    white-space: nowrap;
+  }
+
+  /* Section 1 */
+
+  .section-1 {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    gap: 20px;
+    margin: 0;
+    padding: 150px 20vw 150px 20vw;
+    text-align: center;
+    background-image: url('images/home-bg-desktop.png');
+    background-size: cover;
+    background-position: center;
+    background-repeat: no-repeat;
+  }
+
+  .section-1 .hello {
+    display: none;
+  }
+
+  .section-1 .h1 {
+    color: #ec5242;
+    font-size: 50px;
+    line-height: 50px;
+    margin-bottom: 0;
+    max-width: 460px;
+  }
+
+  .section-1 .red-line {
+    height: 1px;
+    width: 25px;
+    background: #ec5242;
+  }
+
+  .section-1 .p {
+    background: white;
+    border: 1px solid #e3e2e2;
+    padding: 20px;
+    line-height: 25px;
+    font-size: 12px;
+    color: #9b9b9b;
+    max-width: 715px;
+  }
```

Comment:
- [ ] This keeps getting closer but one of the project requirements is to use the given color swatch in your design and not deviate from it in your implementation.
- Your design:
  ![image](https://github.com/suthish777/capstone__01/assets/104968236/71401826-3a54-4579-af1e-57cc385ad5f6)
- Behance design:
  ![image](https://github.com/suthish777/capstone__01/assets/104968236/814d7f58-1140-47a5-a558-30bbb2063a5f)